### ISSUE TITLE
Endpoint information printed when prisma service is deployed for the first time is incorrect

### ISSUE THREAD

**Issue by marktani** [prismagraphql/prisma#2586]
# Bug Report

**Current behavior**

I could only reproduce this when using the following project setup:

```
.
├── .graphqlconfig.yml
├── database
│   ├── datamodel.graphql
│   └── prisma.yml
```

`prisma.yml` has no `endpoint` property.

In the project folder, run `prisma deploy`. Choose any service name and stage name. 

The `endpoint` property in `prisma.yml` is correctly written:

![image](https://user-images.githubusercontent.com/1780597/41040304-54c315a6-699c-11e8-91c8-b36eafe9c3b0.png)

However, at the very end of the output, the endpoint is wrong:

![image](https://user-images.githubusercontent.com/1780597/41040323-63b552fe-699c-11e8-886f-311ecc079ded.png)

**Expected behavior?**

The endpoint contains the workspace slug.

**Comment by marktani** [comment]
I assume this error also causes problems when seeding a project for the first deploy. Then I'll get this error message, and the seeding doesn't succeed:

```graphql
ERROR: Project not found: 'permissions-with-shield-2@dev'

{
  "errors": [
    {
      "message": "Project not found: 'service-name@dev'",
      "code": 3016,
      "requestId": "eu1:api:cji5wl1w8dekp0a96rbxg86xm"
    }
  ],
  "status": 200
}
```

**Comment by divyenduz** [comment]
Fixed via https://github.com/prismagraphql/prisma/pull/2622. I can confirm that this also fixes the seeding issue as mentioned by @marktani

**Comment by divyenduz** [comment]
This is fixed in CLI version 1.10.1. Please use `npm install -g prisma` to update 🎉
